cmd/demo/node.go

```diff
@@ -303,6 +320,33 @@ func findConfig(id wallet.Address) (string, *netConfigEntry) {
 	return "", nil
 }
 
+type UpdateHandler node
```

Comment:
```suggestion
type updateHandler node
```
no need to make it public.